### DISCUSSION TITLE
Integrating Game Creation Features into Toonsquid for Enhanced Interactivity and Web Compatibility

### DISCUSSION THREAD

**Discussion by NikitaKonkolsky** [keiwando/toonsquid#601] [Ideas]
I have a neat idea for Toonsquid that you might find interesting to take into account. What if it was possible to make games with Toonsquid? Like a more updated version of Flash, allowing content to be interactive and exportable as HTML for use in web browsers. Create two-dimensional point-and-click games, or FNAF clones using iPad animation tools, perhaps.

BTW the following image is me if this feature existed.
![THE-FUTURE](https://github.com/keiwando/toonsquid/assets/172333452/1813eeff-67de-45ef-b5f8-9c32fc32c551)

**Comment by CowlBlue** [comment]
As cool as this sounds, Id rather see toonsquid be as flexible and powerful as it can for animation first and foremost!
